Core/SpeckleAutomate/SpeckleAutomate.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
net6 or netstandard2.0 may be preferred.
We should discuss before pushing any nuget.

Reply:
Discussed with Jedd and my take is that we should prefer netstandard2.0 unless higher is required

Reply:
Changed :)